### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by meokbodizi** [dixudx/tumblr-crawler#85]
1. It needs to support basic form of content address like "https://tumblr.blahblah.com/blah"
When approach to a certain tumblr blog with http protocol is blocked by ISP, try https:// or make it as basic form.

2. There should be a method to suppress repeating download when the download fails once.
Save dummy file with the file name, for example.

3. When the address is form of "https://www.tumblr.com/dashboard/blog/blah", it skips downloads.

**Comment by dixudx** [comment]
@meokbodizi Good suggestions. I don't have too much bandwidth to implement this, although this is not very hard. Would you like to help on this? Thanks.

**Comment by meokbodizi** [comment]
Let me try to find the way. Thanks for the fast response.

**Comment by momoe** [comment]
I agree with suggestion 2 the most. Having the ability to use something like an .sqlite database to manage what content has already been downloaded and skip them while processing the tumblr lists would be fantastic! +1 vote.
